### PR TITLE
New package: foldynl.QLog version 0.29.2

### LINKED ISSUE #127325: [Package Request]: QLog
### How can we help?

I would like someone else to build the manifest.

### Please read and ensure the following

- [X] The installer meets the above requirements

### Please provide the following information

Download Page Url: https://github.com/foldynl/QLog/releases
Publisher: Ladislav Foldyna
Package Name: QLog
Description: Amateur radio logbook software.
Package Version: 0.29.2
Installer URL: https://github.com/foldynl/QLog/releases/download/v0.29.2/qlog-installer0.29.2.exe

### PR DESCRIPTION
**Pull request by Dragon1573**
## Description

- Resolve https://github.com/microsoft/winget-pkgs/issues/127325

## Checks

- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/127415)

### CODE REVIEW

File: manifests/f/foldynl/QLog/0.29.2/foldynl.QLog.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+Dependencies:
+  PackageDependencies:
+  - PackageIdentifier: Microsoft.VCRedist.2015+.x64
+AppsAndFeaturesEntries:
+- Publisher: OK1MLG
+  DisplayVersion: 0.29.1
+  ProductCode: '{ff960799-b4b6-4bc6-8b55-0576e9030a78}'
+Installers:
+- Architecture: x64
+  InstallerUrl: http://matebook-x-pro/Programs/qlog-installer0.29.2.exe
```
Comment (Dragon1573) [inline manifests/f/foldynl/QLog/0.29.2/foldynl.QLog.installer.yaml:23]:
```suggestion
  InstallerUrl: https://github.com/foldynl/QLog/releases/download/v0.29.2/qlog-installer0.29.2.exe
```

**Comment by Dragon1573** [comment]
## Notes

Manual validate in Windows Sandbox lead to a warning. Cannot figure out if it can be reproduced.

Snipaste_2023-11-20_23-32-07

**Comment by AsciiWolf** [comment]
You can probably safely ignore this warning if QLog works fine after clicking the "OK" button. It may be a temporary backend issue.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**